tsconfigbase.json

```diff
@@ -7,7 +7,7 @@
     "esModuleInterop": true,
     "incremental": true,
     "jsx": "react",
-    "lib": ["DOM", "DOM.Iterable", "ES2018", "ES2020.Intl"],
+    "lib": ["DOM", "DOM.Iterable", "ES2020", "ES2020.Intl"],
```

Comment:
Probably worth surfacing to PR description. Or opening an issue about changing `target` to `ES2020`.

Reply:
yeah, it's this business:

```
> @jupyterlab/metapackage@4.4.0-alpha.0 build
> tsc -b

../../node_modules/type-fest/source/typed-array.d.ts:16:4 - error TS2583: Cannot find name 'BigInt64Array'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2020' or later.

16  | BigInt64Array
      ~~~~~~~~~~~~~

../../node_modules/type-fest/source/typed-array.d.ts:17:4 - error TS2583: Cannot find name 'BigUint64Array'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2020' or later.

17  | BigUint64Array;
      ~~~~~~~~~~~~~~

```

Reply:
yerp. isolating the change to just `mermaid/{package,tsconfig}.json` doesn't work, either.

I don't think having access to _types_ from the modern specs is going to hurt the generated runtime code, as the `target` is still the same and will complain if there is something it can't downlevel.

Reply:
Yep, this worked.

Reply:
added to the description